packages/vsce/__tests__/__e2e__/10_CheckIfJVMServerISAddedToTheTree.test.ts

```diff
@@ -0,0 +1,74 @@
+import { expect } from "chai";
+import { DefaultTreeSection, SideBarView, TreeItem } from "vscode-extension-tester";
+import { WIREMOCK_PROFILE_NAME, CICSEX61, REGIONS, REGIONS_LOADED } from "./util/constants";
+import {
+  sleep,
+} from "./util/globalMocks";
+import {
+  getCicsSection,
+  getPlexChildIndex,
+  getPlexChildren,
+  getRegionIndex,
+  getRegionResourceIndex,
+  getRegionsInPlex,
+  openZoweExplorer,
+} from "./util/initSetup.test";
+import { resetAllScenarios } from "./util/resetScenarios";
+
+describe("Check if JVM server is present in the tree", () => {
```

Comment:
double check this description is a similar length to others, to make sure the screenshot filename length isn't too big!